lib/mentionsPlugin.js

```diff
@@ -7,7 +7,7 @@ const isLinkOrEmail = require('./util/isLinkOrEmail');
 const mentionRegex = /^@[\w-]+/;
 const whiteSpaceRegex = /\s+/;
 
-function partOfLinkOrEmail (pretext, text, md) {
+function partOfLinkOrEmail (mention, pretext, text, md) {
```

Comment:
Perhaps use the same parameter names as the variable names in the calling context to improve continuity.

Reply:
Make sense--will do.